README.md

```diff
@@ -39,9 +39,17 @@ const server = new ApolloServer({
   resolvers,
 });
 
+
+//Set up our options (optional)
+const options = {
+  middleware: [], //More on this below
+  context: {} // We can pass in our ApolloSever context here
```

Comment:
Context here can't be an empty object, even as a placeholder. `async () => ({})` is the minimum that could be set.